internal/balancergroup/balancergroup_test.go

```diff
@@ -514,6 +571,58 @@ func (s) TestBalancerExitIdleOne(t *testing.T) {
 	}
 }
 
+func (s) TestBalancerGroup_ExitIdleOne_AfterClose(t *testing.T) {
+	balancerName := "stub-balancer-test-exit-idle-one-after-close"
+	called := false
+
+	stub.Register(balancerName, stub.BalancerFuncs{
+		ExitIdle: func(_ *stub.BalancerData) {
+			called = true
```

Comment:
```suggestion
			close(exitIdleCh)
```
Here instead of a variable , a channel can be closed and waited for in a select block since this operation is done only once.